### PR TITLE
Math magicians: full website

### PR DESCRIPTION
**Pull request by rabayarabu**
- Use React Router.
- Use styles in a React app.

### CODE REVIEW

File: src/components/Quotes.js [hunk 1]
```diff
@@ -44,7 +44,7 @@ function Quotes() {
   return (
     <div className="qoute-cont">
       <p>{quoteData && quoteData.quote}</p>
-      <h2>{quoteData.author}</h2>
+      <h2 className="aouthor">{quoteData.author}</h2>
     </div>
   );
```
Comment (shella12) [inline src/components/Quotes.js]: - [ ] Please make sure that the general layout of your `Quotes` page matches the given wireframe in the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md).  As you can see in the wireframe the quote is center aligned

|Your design |wireframe design|
|------|------|
|Screenshot 2023-06-14 at 9 30 01 AM|Screenshot 2023-06-14 at 9 30 26 AM|

**NOTE:** Students are encouraged to add their styles to improve the look and feel.

File: src/components/Home.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  <section>
+    <div className="container">
+      <h2>Welcome to the math world!</h2>
+      <p>Lets play around numbers.</p>
+    </div>
+  </section>
+
+);
+
+export default Home;
```
Comment (shella12) [inline src/components/Home.js]: - [ ] Please make sure that the general layout of your `Quotes` page matches the given wireframe in the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md).  As you can see in the wireframe the home page is left aligned at begin from the start of the page.

|Your design |wireframe design|
|------|------|
|Screenshot 2023-06-14 at 9 42 15 AM|Screenshot 2023-06-14 at 9 30 55 AM|

**NOTE:** Students are encouraged to add their styles to improve the look and feel.

**Review by shella12 — CHANGES_REQUESTED** [review]
Hi @rabayarabu, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- Website has 3 pages: `Home,` `Calculator`, and `Quote` ✅ 
- Dedicated route for each page ✅   
- Navigation links for all three routes ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by amejid — APPROVED** [review]
Hi @rabayarabu, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Linter checks passing :heavy_check_mark: 
- Correct workflow :heavy_check_mark: 
- React router used for navigation :heavy_check_mark: 
- Changes implemented :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**